_posts/2024-09-04-outreachy2024_adetutudeborah.md

```diff
@@ -0,0 +1,65 @@
+
+The goal of this project is to develop a comprehensive social media and communications strategy to grow the MDAnalysis user and contributor base. This involves conducting surveys with MDAnalysis users and developers to understand which communication channels they prefer to use to seek information about and engage with the MDAnalysis project. It also entails finding ways to boost participation and involvement from everyone in the MDAnalysis community and to also implement the setup of the MDAnalysis newsletter.
+
+## What I Did
+
+At the start of the internship, I was able to explore MDAnalysis communication channels and document my observations, recommendations, and current analytics for each channel. I conducted keywords analysis of the MDAnalysis mailing list, Discord, and GitHub discussions along with their frequencies to gain insights into common questions and user needs within the MDAnalysis community. I delved into understanding how participants discover MDAnalysis workshops by gathering data from past MDAnalysis workshops. I also created a methods and definitions document to clarify the metrics used in my report and my approach to the tasks.
+
+Moving forward, I drafted a survey plan, starting with a description of the MDAnalysis target audience and developing research questions to understand what we aim to learn from the survey. From the research questions, I developed the survey questions and answer options, and the Google form to collect data that will inform our communications strategy. Also, I created a survey outreach plan detailing the outreach and advertising channels we plan to use for distributing the survey, the survey participation plan, and the promotional content (blog post, email, and social media post).
+
+We launched the survey form on August 1st and I sent follow-up reminders on August 15th and August 23rd respectively. While waiting for the survey responses, I developed a content strategy which included the proposed content plan and calendar for MDAnalysis. In the last week of the internship, I analyzed and wrote a summary of the survey results and how it can inform MDAnalysis communication strategy.
+
+## The Current State
+
+Currently, the survey results have been analyzed, and a summary has been prepared to inform the MDAnalysis communication strategy and a proposed content plan and calendar for MDAnalysis have been created.
+
+## What's Left to Do
+
+The next steps involve implementing the communications strategy based on the survey results and making adjustments to the strategy as needed. The survey results will guide the selection of communication channels and content types that will best engage the MDAnalysis community. Additionally, launching the MDAnalysis newsletter is another important part of the project. Beyond the internship, an ongoing effort will be needed to sustain the implementation of the communications strategy to grow MDAnalysis user and contributor’s base.
+
+## Links to Project Deliverables
```

Comment:
I think we should delete this whole section, as the blog post is the only thing that has been (and really should be at this point) widely shared with the community.

Reply:
Alright. Noted